src/app/components/navigation/navigation.component.html

```diff
@@ -33,7 +33,7 @@
           <span class="pl-3">Compute Resources</span>
         </a>
       </div>
-      <a mat-list-item (click)="minorComponentsCollapse.show(); executionEnvironmentCollapseContent.hide()">
+      <a mat-list-item (click)="minorComponentsCollapse.show(); executionEnvironmentCollapseContent.hide(); checkPatternAtlasFeatureMode()">
```

Comment:
Couldn't you move all these method calls to a method? I guess, it'll be easier to read if you'd create a separate method for this click event.

Reply:
As you wrote, new method not required anymore.